From 3bf554da8ca6edab25a8662acb8be0bf96e93871 Mon Sep 17 00:00:00 2001
From: Michal Suchanek <msuchanek@suse.de>
Date: Tue, 30 Mar 2021 21:06:10 +0200
Subject: [PATCH] hcn-init.service: Start together with NetworkManager.

hcn-init uses NetworkManager so it does not make sense to run it when
different connection manager is in use. Also when NetworkManager is in
use it should be started automatically.

Signed-off-by: Michal Suchanek <msuchanek@suse.de>
---
 systemd/hcn-init.service.in | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/systemd/hcn-init.service.in b/systemd/hcn-init.service.in
index d0aa2adf7522..768a69e0084b 100644
--- a/systemd/hcn-init.service.in
+++ b/systemd/hcn-init.service.in
@@ -1,11 +1,12 @@
 [Unit]
 Description=hybrid virtual network scan and config
-After=network-online.target
-Wants=network-online.target
+After=NetworkManager.service
+Requisite=NetworkManager.service
+PartOf=NetworkManager.service
 
 [Service]
 Type=oneshot
 ExecStart=@sbindir@/hcnmgr -s
 
 [Install]
-WantedBy=multi-user.target
+WantedBy=NetworkManager.service
-- 
2.26.2

